code/game/objects/items/stacks/stack.dm

```diff
@@ -84,7 +84,7 @@
 				continue
 			t1+="<br>"
 			var/max_multiplier = round(src.get_amount() / R.req_amount)
-			var/title as text
```

Comment:
was this invalid before?

Reply:
the "as text" has no use in local variables.